### PR TITLE
Document MDX docs template

### PR DESCRIPTION
**Pull request by shilman**
Closes N/A

## What I did

Clarified why MDX-based autodocs templates are useful and how to use them

## How to test

Follow the instructions

### CODE REVIEW

**Review by tmeasday — APPROVED** [review]
LGTM

File: docs/writing-docs/autodocs.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Going over the code snippet in more detail. When Storybook starts up, it will ov
 
 #### With MDX
 
-Nevertheless, not all projects are built by adopting a single convention. There may be some situations where you want to use MDX to generate the documentation template. In that case, you can customize the template by adding the [`Meta`](../api/doc-block-meta.md) Doc Block and supplying the `isTemplate` property to tell Storybook to treat this file as a documentation template. For example:
+You can also use MDX to generate the documentation template. This is useful in non-React projects where JSX-handling is not configured. Normally, when you create an MDX file in your project, it is treated as normal documentation. To indicate that an MDX file is a documentation template, supply an the `isTemplate` property to its [`Meta`](../api/doc-block-meta.md) Doc Block. For example:
```
Comment (jonniebigodes) [inline docs/writing-docs/autodocs.md:86]: One small tweak here, if you could remove the `an` after supply, turning this into:
"You can also use MDX to generate the documentation template. This is useful in non-React projects where JSX handling is not configured. Normally, when you create an MDX file in your project, it is treated as normal documentation. To indicate that an MDX file is a documentation template, supply the `isTemplate` property to its [`Meta`](../api/doc-block-meta.md) Doc Block. For example:"

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@shilman left one small item and this should be good to go. Thanks for the clarification.

File: docs/writing-docs/autodocs.md (hunk 1 above)
Comment (shilman) [reply docs/writing-docs/autodocs.md]: Good catch! Fixed.

**Pull request merged**